debian/autobake-deb.sh

```diff
@@ -6,87 +6,17 @@
-  sed '/Package: mariadb-plugin-spider/,/^$/d' -i debian/control
-  sed '/Package: mariadb-plugin-oqgraph/,/^$/d' -i debian/control
-  export MYSQL_COMPILER_LAUNCHER=ccache
-  sed 's|-DDEB|-DPLUGIN_TOKUDB=NO -DPLUGIN_MROONGA=NO -DPLUGIN_SPIDER=NO -DPLUGIN_OQGRAPH=NO -DPLUGIN_PERFSCHEMA=NO -WITH_EMBEDDED_SERVER=OFF -DDEB|' -i debian/rules
-fi
-
+export DEB_BUILD_OPTIONS="nocheck"
 
 # Look up distro-version specific stuff
 #
 # Always keep the actual packaging as up-to-date as possible following the latest
 # Debian policy and targetting Debian Sid. Then case-by-case run in autobake-deb.sh
 # tests for backwards compatibility and strip away parts on older builders.
 
-# If iproute2 is not available (before Debian Jessie and Ubuntu Trusty)
-# fall back to the old iproute package.
-if ! apt-cache madison iproute2 | grep 'iproute2 *|' >/dev/null 2>&1
-then
- sed 's/iproute2/iproute/' -i debian/control
-fi
-
-# If libcrack2 (>= 2.9.0) is not available (before Debian Jessie and Ubuntu Trusty)
-# clean away the cracklib stanzas so the package can build without them.
-if ! apt-cache madison libcrack2-dev | grep 'libcrack2-dev *| *2\.9' >/dev/null 2>&1
-then
-  sed '/libcrack2-dev/d' -i debian/control
-  sed '/Package: mariadb-plugin-cracklib/,/^$/d' -i debian/control
-fi
-
-# If libpcre3-dev (>= 2:8.35-3.2~) is not available (before Debian Jessie or Ubuntu Wily)
-# clean away the PCRE3 stanzas so the package can build without them.
-# Update check when version 2:8.40 or newer is available.
-if ! apt-cache madison libpcre3-dev | grep 'libpcre3-dev *| *2:8\.3[2-9]' >/dev/null 2>&1
-then
-  sed '/libpcre3-dev/d' -i debian/control
-fi
-
-# If libsystemd-dev is not available (before Debian Jessie or Ubuntu Wily)
-# clean away the systemd stanzas so the package can build without them.
-if ! apt-cache madison libsystemd-dev | grep 'libsystemd-dev' >/dev/null 2>&1
-then
-  sed '/dh-systemd/d' -i debian/control
-  sed '/libsystemd-dev/d' -i debian/control
-  sed 's/ --with systemd//' -i debian/rules
-  sed '/systemd/d' -i debian/rules
-  sed '/\.service/d' -i debian/rules
-  sed '/galera_new_cluster/d' -i debian/mariadb-server-10.2.install
-  sed '/galera_recovery/d' -i debian/mariadb-server-10.2.install
-  sed '/mariadb-service-convert/d' -i debian/mariadb-server-10.2.install
-fi
-
-# Convert gcc version to numberical value. Format is Mmmpp where M is Major
-# version, mm is minor version and p is patch.
-# -dumpfullversion & -dumpversion to make it uniform across old and new (>=7)
-GCCVERSION=$(gcc -dumpfullversion -dumpversion | sed -e 's/\.\([0-9][0-9]\)/\1/g' \
-                                                     -e 's/\.\([0-9]\)/0\1/g'     \
-						     -e 's/^[0-9]\{3,4\}$/&00/')
-# Don't build rocksdb package if gcc version is less than 4.8 or we are running on
-# x86 32 bit.
-if [[ $GCCVERSION -lt 40800 ]] || [[ $(arch) =~ i[346]86 ]]
+# Don't build rocksdb package if gcc version is less than 4.8, but everything is,
+# or we are running on x86 32 bit.
```

Comment:
Comment is not correct anymore as it's not build if we are not on x86 32 bit. There is no check for GCC.